### ISSUE TITLE
TS 1.10 Azure F5Telemetry_event_CL does not exist.

### ISSUE THREAD

**Issue by Mikej81** [F5Networks/f5-telemetry-streaming#39]
When using the Azure Log Analytic Consumer, there does not seem to be a F5Telemetry_event_CL.  The example report includes this as the base for the report queries. The other fields do not exist in any of the other tables.

```sql
F5Telemetry_event_CL
| where event_source_s == "request_logging"
|summarize AggregatedValue = count() by bin(TimeGenerated, 1h)
| sort by TimeGenerated desc
```
OR

```sql
F5Telemetry_event_CL
| where isnotempty(attack_type_s)
|summarize AggregatedValue = count() by bin(TimeGenerated, 1h)
| sort by TimeGenerated desc
```

The event type exists in the [constants.js:98](https://github.com/F5Networks/f5-telemetry-streaming/blob/556cd01de7686fb03b69c603771c3ff624eac726/src/lib/constants.js#L98):

```json
    EVENT_TYPES: {
        DEFAULT: 'event',
        AVR_EVENT: 'AVR',
        APM_EVENT: 'APM',
        ASM_EVENT: 'ASM',
        AFM_EVENT: 'AFM',
        LTM_EVENT: 'LTM',
        CGNAT_EVENT: 'CGNAT',
        SYSLOG_EVENT: 'syslog',
        EVENT_LISTENER: 'event',
        SYSTEM_POLLER: 'systemInfo',
        IHEALTH_POLLER: 'ihealthInfo'
    }
```
### Environment
 * Telemetry Streaming Version: 1.10
 * BIG-IP Version: 14.1.2 (Azure PAYG)

### Summary
Please also include information about the reproducibility and the severity/impact of the issue.

### Steps To Reproduce
1. Submit the following declaration:
```json
{
    "class": "Telemetry",
    "controls": {
        "class": "Controls",
        "logLevel": "info"
    },
    "My_System": {
        "class": "Telemetry_System",
        "systemPoller": {
            "interval": 60
        }
    },
    "My_Listener": {
        "class": "Telemetry_Listener",
        "port": 6514
    },
   "My_Consumer": {
       "class": "Telemetry_Consumer",
       "type": "Azure_Log_Analytics",
       "workspaceId": "blah",
       "passphrase": {
           "cipherText": "secret"
       }
   }
}
```
2.  Import Azure Log Analytic Report Example and view following error:

```
'where' operator: Failed to resolve table or column expression named 'F5Telemetry_event_CL'

If issue persists, please open a support ticket.

Request id: 8f1070d7-4c27-4ac7-8c15-3f5b15294221
```

### Expected Behavior
F5Telemetry_event_CL should exist.

### Actual Behavior
F5Telemetry_event_CL does not exist.

Including a screenshot of all the events that are sending.  None of the events include the fields for attack_type_s or event_source_s , etc.

![Screen Shot 2020-04-03 at 1 01 38 PM](https://user-images.githubusercontent.com/13910667/78387367-352aa280-75ad-11ea-897f-49182bd5c50c.png)

**Comment by dstokesf5** [comment]
Do you have any BIG-IP features that are using your TS event listener as a logging destination?
https://clouddocs.f5.com/products/extensions/f5-telemetry-streaming/latest/event-listener.html

**Comment by Mikej81** [comment]
Configured in ASM and AFM.  Waiting on AUTOTOOL-768 to spool up again and test.

**Comment by dstokesf5** [comment]
It looks like the example is outdated and the F5Telemetry_event records have now been broken up into much more specific event types (e.g. F5Telemetry_LTM). I have added AUTOTOOL-1503 to our internal product backlog to update the example.

**Comment by dstokesf5** [comment]
This issue has been resolved as part of the 1.13.0 release.
